src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -37,6 +37,7 @@
     <InternalsVisibleTo Include="InteractiveHost" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.EditorFeatures" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.EditorFeatures.Wpf" />
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.Razor" />
```

Comment:
why razor? debugger memory tool is not razor team? or find all reference seems very generic, is this the pattern we are going to have? every team implementing same thing in their IVT dll?

Reply:
> why razor? debugger memory tool is not razor team?

My understanding from Marco is the Razor team owns this assembly.

> is this the pattern we are going to have? every team implementing same thing in their IVT dll?

Yes. Each team owns their adapter assembly and can make changes to its APIs without consulting other teams. Sharing implementations between teams defeats almost all of the advantages of using these adapters. Features general enough to share can be moved to public APIs, removing the need for any duplication.

Reply:
I am sorry, I just understood the issue. Razor is the internal name that we use for the VsDebugPresentationPackage. I am part of the debugger team and we own that dll. I did not know there was a razor team.

Reply:
@sharwell sure. but the code seems really generic. might be better to have this code in Roslyn and share the code here and in OB? 
http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs,530

and here, it should call the API rather than having all the plumbing done again here where Object brower basically has same thing?

Reply:
This should probably change to Microsoft.CodeAnalysis.ExternalAccess.VsDebugPresentationPackage

Reply:
Now updated